app/swagger/schemas/async_transaction/vet360.rb

```diff
@@ -32,6 +32,11 @@ class Vet360
             end
           end
         end
+
+        swagger_schema :AsyncTransactionsVet360 do
+          key :required, [:data]
```

Comment:
Is this good enough? Is there a way to reference the single tx response schema for the items of the array?